src/components/form/select/select.tsx

```diff
@@ -197,21 +210,23 @@ export const Select: React.FC<SelectProps> = ({
         })}
         tabIndex={0}
         ref={refs.setReference}
-        aria-labelledby="select-label" // fixme
+        title={label || undefined}
         aria-autocomplete="none"
         {...getReferenceProps()}
         {...props}
       >
         {/* This is here for compatibility with native forms, as well as a providing a target for change events. */}
         <select
+          id={id}
           ref={fakeInputRef}
           name={name}
           defaultValue={selectedOptionValue}
           hidden={true}
+          aria-label=""
         >
-          {selectedOptionValue && (
+          {(selectedOptionValue && (
```

Comment:
A select can also use `<optgroup label = "Choose schoolyear">` to clarify what type of stuff can be selected, instead of using a Div.